aws_tools/s3_handler.py

```diff
@@ -95,19 +94,10 @@ def upload_file(self, path, key, cache_time=600):
     def get_object(self, key):
         return self.resource.Object(bucket_name=self.bucket_name, key=key)
 
-    def get_contents(self, key, catch_exception=True):
```

Comment:
I made sure nothing used the get_contents() function. It's the identical get_file_contents() function that everything uses.